builtin/game/chatcommands.lua

```diff
@@ -225,6 +225,19 @@ core.register_chatcommand("revoke", {
 			end
 		end
 		core.set_player_privs(revoke_name, privs)
+
+		--Check if we succeeded
+		privs = core.get_player_privs(revoke_name)
+		local check_privs = revoke_privs
+		if revoke_priv_str == "all" then
+			check_privs = privs
+		end
+		for priv, _ in pairs(check_privs) do
+			if privs[priv] == true then
+				return false, "Privileges of " .. revoke_name .. " could not be revoked.. are they admin?"
```

Comment:
before the return, can you log the attempted revocation of the privileges? Sometimes knowing who tried sth can be interesting ...